docs/en/tutorials/how_to_install_mmdeploy_on_jetsons.md

```diff
@@ -26,102 +26,196 @@ Sometimes we just need to reboot the jetson device when it gets stuck in initial
 ### Cuda
 
 The Cuda is installed by default while the cudnn is not if we use the first method. We have to write the cuda path and lib to `$PATH` and `$LD_LIBRARY_PATH`:
-```
+```shell
 export PATH=$PATH:/usr/local/cuda/bin
 export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/usr/local/cuda/lib64
 ```
 Then we can use `nvcc -V` the get the version of cuda we use.
 
-### Anaconda
+### Conda
 
-We have to install [Archiconda](https://github.com/Archiconda/build-tools/releases) instead as the Anaconda does not provide the wheel built for jetson.
+We have to install [Archiconda](https://github.com/Archiconda/build-tools/releases) instead as the Anaconda does not provide the wheel built for jetson. The commands below are the example for installation, you can choose another version of Archiconda by accessing [Archiconda releases page](https://github.com/Archiconda/build-tools/releases).
+```shell
+wget https://github.com/Archiconda/build-tools/releases/download/0.2.3/Archiconda3-0.2.3-Linux-aarch64.sh
+bash Archiconda3-0.2.3-Linux-aarch64.sh
+source ~/.bashrc
+conda --version
+```
 
-After we installed the Archiconda successfully and created the virtual env correctly. If the pip in the env does not work properly or throw `Illegal instruction (core dumped)`, we may consider re-install the pip manually, reinstalling the whole JetPack SDK is the last method we can try.
+After we installed the Archiconda successfully, we need to check the tensorrt python package pre-installed by Jetpack by command below.
+```shell
+sudo find / -name tensorrt
+```
+Then you can see something like those in the terminal, take Jetson Nano as example:
+```shell
+...
+/usr/lib/python3.6/dist-packages/tensorrt
+...
+```
+The `python3.6` is the version we need to use in the conda env later.
+
+We can create the virtual env `mmdeploy` using the command below. The version of python we got from the previous step.
+```shell
+conda create -n mmdeploy python=x.x
```

Comment:
We should specify the python versio to 3.6 in `conda create` command, shouldn't we?

Reply:
I think is 3.6 for all JetPack. And also the Nvidia privide torch wheel only for 3.6, So I change the `x.x` => `3.6`. 😄